src/stories/ApiDoc.stories.mdx

```diff
@@ -13,10 +13,9 @@ import { Meta } from '@storybook/addon-docs';
     }}
 />
 
-<!-- Include the generated esdoc in an iframe.
-The output directory for esdoc is docs, that is set as static directory in main.js. -->
+<!-- We use parent.window because we are already inside an iframe.-->
 <iframe
-    src="./index-docs.html"
+    src={(new URLSearchParams(parent.window.location.search)).get('subpath') ? (new URLSearchParams(parent.window.location.search)).get('subpath') : 'index-docs.html'}
```

Comment:
in mdx, daca as fi vrut sa pun new URL... intr-o constanta ca sa evit copy/paste-ul, cum s-ar fi putut face? poate definit component, si apoi in "programul principal", `<MyComp />`? 

Adica tot codul (md + jsx) din "programul principal" cred ca e intr-un return al unui component (probabil functional). Om avea posibiliattea sa scriem si cod in acest component in zona "de dinaintea" return-ului?

Reply:
Am lucrat aseara si nu cred ca merge. In "corpul principal", nu pare posibil code deloc.